include/CoreText/CoreTextExport.h

```diff
@@ -1,6 +1,6 @@
 //******************************************************************************
 //
-// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
The can easily be separated into a commit. 

---
In reply to: [112725826](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112725826) [](ancestors = 112725826)